common/hyperscaler/rules/sorter.go

```diff
@@ -0,0 +1,19 @@
+package rules
+
+import "sort"
+
+func SortRuleEntries(entries []ParsingResult) []ParsingResult {
```

Comment:
nit: the number of input attributes could be {0, 1, 2}. We do not restrict number of rules.
From the complexity point of view count sort could be more appropriate.